canonical-web-and-design/www.ubuntu.com
# Update template variables for 19.04

## Done

- Updated the django template variables for the new 'latest' release

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that the download dropdown looks correct for 19.04 and the download section as well.

## Issue / Card

Fixes #4926

## Screenshots

![image](https://user-images.githubusercontent.com/441217/56095972-06b64200-5eda-11e9-9727-31de71a02c66.png)

![image](https://user-images.githubusercontent.com/441217/56095978-16ce2180-5eda-11e9-811a-c573373f640c.png)
